File: 1950-sign-of-the-product-of-an-array/solution.c
int arraySign(int* nums, int numsSize) {
    int nc=0,pc=0;
    for(int i=0;i<numsSize;i++){
        if(nums[i]==0){
            return 0;
        }
        else if(i[nums]<0){
            nc++;
        }
        else{
            pc++;
        }
    }
    if(nc%2!=0){
        return -1;
    }
    return 1;
}

File: 1950-sign-of-the-product-of-an-array/solution.java
class Solution {
    public int arraySign(int[] nums) {
        int n = 0, p = 0;
        int len = nums.length;
        for (int i = 0; i < len; i++) {
            if (nums[i] == 0)
                return 0;
            else if (nums[i] < 0)
                n++;
            else
                p++;
        }
        if (n % 2 != 0)
            return -1;
        return 1;
    }
}
